src/components/playerOverview/MatchPlacementBarChart.tsx

```diff
@@ -20,8 +21,13 @@ export const MatchPlacementBarChart = React.memo(function MatchPlacementBarChart
     }
 
     const matchPlacementDictionary: { [rank: string]: number } = {};
+    const validMatches = filterMatchesByPlayerCount(matches, NUMBER_OF_PLAYERS_FOR_VALID_MATCH);
 
-    for (const match of matches) {
+    if (validMatches === undefined) {
```

Comment:
the intention of this is good! however, if you look into the function filterMatchesByPlayerCount, it will never return undefined. instead, it will return an empty array.

Reply:
Noted. Changed to .len and === 0.

Reply:
Funny thing: if you look at the commanderOverview equivalent you'd notice I actually wrote the correct logic there. I probably noticed my mistake there and forgot to fix the other one. Thanks for noticing!